lighthouse-core/audits/screenshot-thumbnails.js

```diff
@@ -81,12 +83,19 @@ class ScreenshotThumbnails extends Audit {
     const trace = artifacts.traces[Audit.DEFAULT_PASS];
     const cachedThumbnails = new Map();
 
-    return artifacts.requestSpeedline(trace).then(speedline => {
+    return Promise.all([
+      artifacts.requestSpeedline(trace),
+      TTFI.audit(artifacts).catch(() => ({})),
+      TTCI.audit(artifacts).catch(() => ({})),
+    ]).then(([speedline, ttfi, ttci]) => {
       const thumbnails = [];
       const analyzedFrames = speedline.frames.filter(frame => !frame.isProgressInterpolated());
+      const ttfiVal = typeof ttfi.rawValue === 'number' ? ttfi.rawValue : 0;
```

Comment:
maybe move `rawValue: 0` into the catch statements above so it's clear they're fallbacks only for that case?